package.json

```diff
@@ -52,14 +54,15 @@
     "bit-twiddle": "^1.0.2",
     "earcut": "^2.0.7",
     "eventemitter3": "^2.0.0",
-    "glslify": "^5.0.2",
     "ismobilejs": "^0.4.0",
     "object-assign": "^4.0.1",
     "pixi-gl-core": "^1.0.3",
     "resource-loader": "^1.8.0",
     "browserify-versionify": "^1.0.6"
```

Comment:
We use this to replace the version string, I assume that doesn't happen in the ES5 files. How can we inject the string into those files (before browserify runs).